pkg/rescheduler/strategies/duplicates.go

```diff
@@ -47,33 +57,37 @@ func RemoveDuplicatePods(client clientset.Interface, strategy api.ReschedulerStr
 					//fmt.Printf("Removing duplicate pod %#v\n", k.Name)
 					success, err := evictions.EvictPod(client, pods[i], policyGroupVersion)
 					if !success {
+						//TODO: change fmt.Printf as glogs.
 						fmt.Printf("Error when evicting pod: %#v (%#v)\n", pods[i].Name, err)
 					} else {
+						podsEvicted++
 						fmt.Printf("Evicted pod: %#v (%#v)\n", pods[i].Name, err)
 					}
 				}
 			}
 		}
 	}
+	return podsEvicted
 }
 
-func RemoveDuplicatePodsOnANode(client clientset.Interface, node *v1.Node) DuplicatePodsMap {
+// ListDuplicatePodsOnANode lists duplicate pods on a given node.
+func ListDuplicatePodsOnANode(client clientset.Interface, node *v1.Node) DuplicatePodsMap {
 	pods, err := podutil.ListPodsOnANode(client, node)
 	if err != nil {
 		return nil
 	}
 	return FindDuplicatePods(pods)
 }
 
+// FindDuplicatePods takes a list of pods and returns a duplicatePodsMap.
 func FindDuplicatePods(pods []*v1.Pod) DuplicatePodsMap {
 	dpm := DuplicatePodsMap{}
-
 	for _, pod := range pods {
 		sr, err := podutil.CreatorRef(pod)
 		if err != nil || sr == nil {
 			continue
 		}
-		if podutil.IsMirrorPod(pod) || podutil.IsDaemonsetPod(sr) || podutil.IsPodWithLocalStorage(pod) {
+		if podutil.IsMirrorPod(pod) || podutil.IsDaemonsetPod(sr) || podutil.IsPodWithLocalStorage(pod) || podutil.IsCriticalPod(pod) {
```

Comment:
Its failing merge because of this, can you rebase?

Reply:
I see that there are no conflicts atleast in the github UI?